extras/rxjava2/src/main/java/org/asynchttpclient/extras/rxjava2/DisposedException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2017 AsyncHttpClient Project. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ *     http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.asynchttpclient.extras.rxjava2;
+
+import java.util.concurrent.CancellationException;
+
+/**
+ * Indicates that the HTTP request has been disposed asynchronously via RxJava.
+ */
+public class DisposedException extends CancellationException {
+    private static final long serialVersionUID = -2913992117415097629L;
+
+    public DisposedException() {
+    }
+
+    public DisposedException(final Throwable cause) {
```

Comment:
Yep, just like in the RxJava1 extras. It's just a convenience thingy when someone wants to use/extend this stuff. I'll remove it if you prefer...?